voluncheer/profiles/management/commands/load_badge_data.py

```diff
@@ -0,0 +1,50 @@
+from datetime import timedelta
+import json
+import re
+
+from django.core.management.base import BaseCommand
+
+from profiles.models import Badge
+
+regex = re.compile(r"((?P<hours>\d+):)((?P<minutes>\d+):)(?P<seconds>\d+)")
```

Comment:
Is there any advantage to manually implement load_badge_data.py? Doesn't Django take care of it for you automatically with load_data json files in fixtures directory? Did I misunderstood sth?

Reply:
The advantage is that this will run on deployment  automatically without having to go in and manually loading the fixture from the EB console. Again, just in case we need to rebuild the db. It will not create badges again if they already exist.